README.md

```diff
@@ -66,4 +66,28 @@ It is also possible to read GOB events from a batch file with line-separeted eve
 
     schema import events <path-to-dataset> <path-to-file-with-events>
 
-
+## Schema Tools as a pre-commit hook
+
+Included in the project is a `pre-commit` hook
+that can validate schema files
+in a project such as [amsterdam-schema](https://github.com/Amsterdam/amsterdam-schema)
+
+To configure it
+extend the `.pre-commit-config.yaml`
+in the project with the schema file defintions as follows:
+
+```yaml
+  - repo: https://github.com/Amsterdam/schema-tools
+    rev: v0.18.1
+    hooks:
+      - id: validate-schema
+        args: ['https://schemas.data.amsterdam.nl/schema@v1.1.1#']
+        exclude: 'datasets/index.json$'
```

Comment:
When working on a branch of schema-tools, using an new dataset, still under development, this dataset is not yet at the production location (the `args`). Can this location be made configurable in some way?

Reply:
That's what the `args` command is for. You can change its value in `.pre-commit-config.yaml ` of the `amsterdam-schema` repo.

But the fact that we need it at all confuses me. Each schema already defines the meta schema it adheres to. Why then does the existing `validate` command, and hence the derived command `batch-validate`, require the meta schema to be explicitly specified?

Reply:
Nevermind, because `jsonschema.validate` works like that.

Should we add some sort of preprocessing to it (in a different PR) so that it can select the meta schema from the schema, obviating the need for us to specify it explicitely?

Reply:
Resolving as the `args` parameter can be adjusted. But this definitely needs more discussion.